package.json

```diff
@@ -5,7 +5,6 @@
   "main": "Universal-Federated-Analytics.js",
   "type": "module",
   "scripts": {
-    "build": "npx uglifyjs Universal-Federated-Analytics.js --output Universal-Federated-Analytics-Min.js --source-map url=Universal-Federated-Analytics-Min.js.map",
```

Comment:
This script is misleading. It makes people (including me, originally) think that the DAP code is minified with `uglifyjs`. In reality, CP uses the [Closure Compiler](https://developers.google.com/closure/compiler) for minification.